src/main/java/com/kodilla/ecommercee/domain/CartDto.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.ecommercee.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartDto {
```

Comment:
Klasa już istnieje

Reply:
Rebase zrobiony

Reply:
ale ja dalej widzę to musisz spróbować jeszcze raz

Reply:
Potwierdzam, zmiany dotyczące CartDto dalej widać